app/src/main/java/com/example/javacodersnairobi/adapter/GithubAdapter.java

```diff
@@ -9,20 +9,21 @@
 import android.widget.ImageView;
 import android.widget.TextView;
 
+import com.example.javacodersnairobi.MainActivity;
 import com.example.javacodersnairobi.R;
 import com.example.javacodersnairobi.model.GithubUsers;
 import com.squareup.picasso.Picasso;
 
 import java.util.List;
 
-public class GithubAdapter extends RecyclerView.Adapter<GithubAdapter.GithubUsersViewHolder> {
+public class GithubAdapter extends RecyclerView.Adapter<GithubAdapter.MyViewHolder> {
     private List<GithubUsers> dataSet;
     private Context context;
 
-    public static class GithubUsersViewHolder extends RecyclerView.ViewHolder {
+    public static class MyViewHolder extends RecyclerView.ViewHolder {
```

Comment:
Why did you have to change this class' name to "MyViewHolder", I feel it isn't descriptive enough. Isn't the old name just fine? "GithubUsersViewHolder" or "UsersViewHolder". Please find a more suitable name.

Reply:
Thanks for this observation @damafeez, I have made the corresponding adjustment.